hw12_13_14_15_calendar/internal/config/config.go

```diff
@@ -1,84 +1,48 @@
 package config
 
 import (
-	"context"
-	"errors"
-
-	"github.com/heetch/confita"
-	"github.com/heetch/confita/backend/env"
-	"github.com/heetch/confita/backend/file"
+	"github.com/ilyakaznacheev/cleanenv"
```

Comment:
I had to migrate to this package, because of problems with `confita` https://github.com/heetch/confita/issues/77